Enhancer/EnhancerInterface.php

```diff
@@ -28,8 +27,8 @@
      *
      *     $context->addData('foobar', 'Some value');
      *
-     * @param Context Serialization context
-     * @param resource The resource being serialized
+     * @param []       $data        Context Serialization context
+     * @param Resource $resource    The resource being serialized
      */
-    public function enhance(array $data, PuliResource $resource);
+    public function enhance(array $data, Resource $resource);
```

Comment:
@WouterJ i did the change here, but i think it is the other way around as you said it. Or do we we have some version proplems in our chain from resource -> resource-bundle -> resource-restbundle ->sandbox?
The major problem is that master on Puli is behind several beta-* branches, otherwise we should use dev-master at all instances to got the same behavior.